### DISCUSSION TITLE
[RocketMQ 5.1.4] 查询集群信息无法查到从节点

### DISCUSSION THREAD

**Discussion by duya23333git** [apache/rocketmq#7553] [Q&A]
集群配置：采取一主一从

问题：开启controller模式后无法查询slave节点信息，其余功能均可正常使用
![image](https://github.com/apache/rocketmq/assets/38960007/c7caa41e-f018-4e34-828e-c13f907ad29f)

相关配置信息：
`brokerClusterName=xxhx-cluster
brokerName=broker-a
brokerId=-1
deleteWhen=04
fileReservedTime=48
brokerRole=SLAVE
flushDiskType=ASYNC_FLUSH
namesrvAddr=192.168.64.123:9876;192.168.64.124:9876
storePathRootDir=/data/rocketmq/data
storePathCommitLog=/data/rocketmq/commitlog

# Controller
enableControllerMode=true
controllerAddr=192.168.64.123:9878;192.168.64.124:9878
allAckInSyncStateSet=true
`

**Comment by duya23333git** [comment]
因上述配置文件错乱，重新发下配置文件信息
``` shell
brokerClusterName=xxhx-cluster
brokerName=broker-a
brokerId=-1
deleteWhen=04
fileReservedTime=48
brokerRole=SLAVE
flushDiskType=ASYNC_FLUSH
namesrvAddr=192.168.64.123:9876;192.168.64.124:9876
storePathRootDir=/data/rocketmq/data
storePathCommitLog=/data/rocketmq/commitlog
# Controller
enableControllerMode=true
controllerAddr=192.168.64.123:9878;192.168.64.124:9878
allAckInSyncStateSet=true
```

**Reply by figuewang** [reply]
-1  改成1

**Reply by duya23333git** [reply]
感谢您的回答，之前是按照主节点设1，从节点设2这样子来做的，但依然出现这种情况，所以就全改成-1了。经测试，我发现它好像仅仅是不在列表内，如果主挂了，controller还是会将从设为主
